assets/stylesheets/landing.css

```diff
@@ -32,25 +43,35 @@ body {
 }
 
 .glitch {
-  font-size: 18vh;
+  color: #f8f8f8;
+  font-size: 15vh;
```

Comment:
dont use `vh` for font-size, use `rem` for font-size, and for root font size, use calc() with fixed pixel unit and add some vw units to make font size responsive and fluid.